### PR TITLE
More cleanup for the skeleton rails app

### PR DESCRIPTION
**Pull request by benthorner**
* Remove app/jobs, vendor/, bin/update, package.json, config/puma.rb
  * Upgrade to ruby 2.5.1
  * Enable rubocop check for FrozenStringLiteralComment
  * Version all gems (including dev/test)
  * Add designated port to Procfile

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: bundle exec unicorn -c ./config/unicorn.rb -p ${PORT}
+web: bundle exec unicorn -c ./config/unicorn.rb -p ${PORT:-3221}
```
Comment (kevindew) [inline Procfile]: Great port 🥇 💯

File: db/migrate/20160622154200_create_users.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```
Comment (kevindew) [inline db/migrate/20160622154200_create_users.rb]: Not really related to this file. But how did we get such an old migration?

File: Gemfile [hunk 3]
```diff
@@ -11,15 +13,15 @@ gem "pg", "~> 1"
...
-  gem "govuk-lint"
-  gem "listen"
+  gem "byebug", "~> 10"
+  gem "govuk-lint", "~> 3"
+  gem "listen", "~> 3"
 end
 
 group :test do
-  gem "simplecov", require: false
+  gem "simplecov", "~> 0.16"
```
Comment (kevindew) [inline Gemfile]: do you mean to strip the require from this?

Comment (benthorner) [reply Gemfile]: Yes - doesn't seem to affect anything...

File: db/migrate/20160622154200_create_users.rb (hunk 2 above)
Comment (benthorner) [reply db/migrate/20160622154200_create_users.rb]: Another gift from that script repo...

**Comment by kevindew** [comment]
Shall we get a quick PR in so we have CI?

**Comment by benthorner** [comment]
I think @tijmenb was doing something there...

**Pull request merged**